runtime/tools/python/ttrt/common/debugger.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import json
+import importlib.machinery
+import sys
+import signal
+import os
+import io
+import subprocess
+import time
+import socket
+from pkg_resources import get_distribution
+import shutil
+import atexit
+import re
+from functools import partial
+
+from ttrt.common.util import *
+
+
+class DebuggerRuntimeConfig:
+    def __init__(self, device=None, artifact_dir=""):
+        self.device = device
+        self.artifact_dir = artifact_dir
+
+
+def debugger(device_runtime_config, binary, program_context, op_context):
+    device = device_runtime_config.device
+
+    print("-----------executing debugger-----------")
+
+    try:
+        import pdb
+
+        pdb.set_trace()
+    finally:
+        print("-----------finished executing debugger-----------")
```

Comment:
I think we should remove these print statements

Reply:
fixed